src/typescript-generator/script.js

```diff
@@ -36,9 +36,7 @@ export class Script {
   }
 
   export(coder, isType = false, isDefault = false) {
-    const cacheKey = isDefault
-      ? "default"
-      : `${coder.id}@${nodePath}:${isType}`;
```

Comment:
`nodePath` was always `[Object: object]`. IIRC that was supposed to be the URL of the requirement, which is now part of `coder.id`.